### PR TITLE
[BACKLOG-11565]: Create and Compile HDP 2.5 SHIM against Pentaho 7.1 …

### PR DESCRIPTION
**Pull request by TatsianaKasiankova**
…(master)

The new HDP 2.5 shim.

### CODE REVIEW

**Comment by TatsianaKasiankova** [comment]
@dkincade , please review. This is the new HDP 2.5 shim - CE version.

File: build.properties [hunk 1]
```diff
@@ -6,4 +6,4 @@ project.revision=7.1-SNAPSHOT
 
 junit.maxmemory=500M
 
-build.modules.list=hive-jdbc,hdp23,hdp24,cdh57,cdh58,mapr410,mapr510,emr310,emr41,emr46
+build.modules.list=hive-jdbc,hdp23,hdp24,hdp25,cdh57,cdh58,mapr410,mapr510,emr310,emr41,emr46
```
Comment (dkincade) [inline build.properties]: Please remove hdp23 from this list

File: hdp25/build.properties [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+common.hadoop.shim.version=1.0
+common.hbase.shim.version=1.1
+common.pig.shim.version=1.1
+hbase.generation=1.0
+
+dependency.kettle.revision=7.1-SNAPSHOT
+dependency.pentaho-hdfs-vfs.revision=7.1-SNAPSHOT
+dependency.oss-licenses.revision=7.1-SNAPSHOT
+
+dependency.xstream.revision=1.4.2
```
Comment (dkincade) [inline hdp25/build.properties]: We need to try and see if we can use 1.4.7 as versions before that have a security vulnerability

File: build.properties (hunk 1 above)
Comment (dkincade) [inline build.properties]: Also, has the build properties that need to be created been reviewed by @buildguy

Comment (TatsianaKasiankova) [inline build.properties]: @dkincade do you mean this property: https://github.com/pentaho/build-resources/blob/master-qat/version.properties#L36?

Comment (TatsianaKasiankova) [inline build.properties]: As I understand version.properties for master branch have not been updated yet for 7.1 .

**Comment by TatsianaKasiankova** [comment]
@dkincade, please see the PR with fixes:

- removed hdp23 from this list;

-  used  "xstream" rev="1.4.9" (http://jira.pentaho.com/browse/COMP-3197);

- dependency.hadoop-shims-hdp25.revision=70-SHIM-QAT-BUILDTAG is already in https://github.com/pentaho/build-resources/blob/master-qat/version.properties#L36

I'd like to note that the library **xstream-1.4.9.jar** is not shipped with the shim but only used on compile time (resolved with provided scope).

**Pull request merged**